lib/fastlane/actions/badge.rb

```diff
@@ -51,7 +51,15 @@ def self.available_options
                                        env_name: "FL_BADGE_SHIELD",
                                        description: "add a shield to your app icon from shield.io",
                                        optional: true,
-                                       is_string: true)
+                                       is_string: true),
+          FastlaneCore::ConfigItem.new(key: :alpha,
+                                       env_name: "FL_BADGE_ALPHA",
+                                       description: "adds and alpha badge instead of the default beta one",
+                                       optional: true,
+                                       is_string: false,
+                                       verify_block: proc do |value|
+                                         raise "alpha is only a flag and should always be true".red unless value == true
```

Comment:
Its just a flag, `false` values are not allowed.
If I add more features in the future I may rethink this architecture.